include/aspect/geometry_model/chunk.h

```diff
@@ -188,6 +188,9 @@ namespace aspect
             push_forward(const Point<dim> &chart_point) const;
         };
 
+        Point<dim> pull_back(const Point<dim>) const;
+        Point<dim> push_forward(const Point<dim>) const;
+
```

Comment:
These lines seem unnecessary for the current pull request, right? Can you remove them?